controllers/nutanixcluster_controller.go

```diff
@@ -44,6 +43,7 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/source"
 
 	infrav1 "github.com/nutanix-cloud-native/cluster-api-provider-nutanix/api/v1beta1"
+	nutanixClient "github.com/nutanix-cloud-native/cluster-api-provider-nutanix/pkg/client"
```

Comment:
>They are lower case, with no under_scores or mixedCaps.

nit: https://go.dev/blog/package-names
```suggestion
	nutanixclient "github.com/nutanix-cloud-native/cluster-api-provider-nutanix/pkg/client"
```